Fix uuid import syntax to use mandatory module syntax

actuallymentor opened the pull request:
# Why

While `uuid@6` prefers named imports `uuid@8` requires it. Using this import gets rid of the `Deep requiring like const uuidv4 = require('uuid/v4'); is deprecated as of uuid@7.x.` message in v7 and makes the constants package compatible with uuid v8.x

# How

Simple import syntax fix.

# Test Plan

Run any web project with constants.

actuallymentor committed:
Fix uuid imort syntax to use mandatory module syntax

ide reviewed (changes requested):
Please rebuild the JS and add it to your commit: https://github.com/expo/expo/blob/master/CONTRIBUTING.md#-before-submitting

actuallymentor committed:
Built expo constants files

actuallymentor committed:
Added changelog message

actuallymentor commented:
I foolishly didn't read the guidelines. Read then just now and:

- [x] Added build files
- [x] Added changelog

If I missed anything let me know.

actuallymentor committed:
Merge branch 'master' into fix-constants-uuid-import

bbarthec reviewed (approved):
LGTM 💯

bbarthec commented on CHANGELOG.md:
```diff
@@ -18,6 +18,7 @@ This is the log of notable changes to the Expo client that are developer-facing.
 
 ### 🐛 Bug fixes
 
+- Fixed `uuid`'s deprecation of deep requiring ([#8114](https://github.com/expo/expo/pull/8114) by [@actuallymentor](https://github.com/actuallymentor) )
```
```suggestion
- Fixed `uuid`'s deprecation of deep requiring ([#8114](https://github.com/expo/expo/pull/8114) by [@actuallymentor](https://github.com/actuallymentor))
```
Move this change to `packages/expo-constants/CHANGELOG` as this affects `expo-constants` module only and not whole repository 😉

actuallymentor committed:
Update CHANGELOG.md

Co-authored-by: Bartłomiej Bukowski <bartlomiejbukowski.b@gmail.com>
